doc/build.md

```diff
@@ -8,7 +8,8 @@
 - zlib libs (zlib1g-dev or zlib-devel)
 - pkg-config
 - linux-headers (reported needed on Alpine linux)
-- llvm
+- llvm (which can be downloaded [here](https://github.com/llvm/llvm-project/releases))
+- cmake
```

Comment:
I'll just suggest that you add here the version of cmake. I had a problem with it that I needed to downgrade.